packages/Confirmation/stories/examples/ConfirmationExample.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Button from "@paprika/button";
+import Confirmation from "../../src";
+
+const ConfirmationExample = () => {
+  const [isConfirmationShowing, setIsConfirmationShowing] = React.useState(false);
+  return (
+    <div>
+      <Button id="triggerElement" isSemantic={false} onClick={() => setIsConfirmationShowing(true)}>
+        Trigger
+      </Button>
+      {isConfirmationShowing ? (
+        <Confirmation
+          isOpenByDefault
+          align="right"
+          buttonSize="medium"
+          confirmLabel="Confirm button text"
+          description="Description text"
+          onConfirm={() => {}}
+          onCancel={() => {}}
+          onClose={() => {}}
+          heading="Delete filter?"
+          getPositioningElement={() => document.getElementById("triggerElement")}
+          offset={4}
```

Comment:
@mikrotron I'm not finding `offset` making a difference in the story example, did you get a chance to look into this usage?

Reply:
I see this having an affect in Storybook.  Because `align='right'`, it creates a gap to the right of the trigger button and the confirmation popover.